solr/core/src/test/org/apache/solr/search/TestCoordinatorRole.java

```diff
@@ -338,6 +338,7 @@ public void testNRTRestart() throws Exception {
         start = new Date().getTime();
         try {
           do {
+            // TODO - What is expected here? equals not doing anything? hostCore=?
```

Comment:
it should be this
```
      if( pullCore.equals(
                    getHostCoreName(
                        COLL,
                        qaJettyBase,
                        client,
                        p -> {
                          p.set(CommonParams.Q, "id:345");
                          p.add("shards.preference", "replica.type:NRT");
                        }))){
              done.set(true);
            }
```

Reply:
Fixed in e409355e572f08491b199841f773166e0b4c7b3c I think.